sqlserver/datadog_checks/sqlserver/utils.py

```diff
@@ -25,6 +25,19 @@ def construct_use_statement(database):
     return 'use [{}]'.format(database)
 
 
+def is_statement_proc(text):
+    if text:
+        t = text.upper().split()
+        idx_create = t.index('CREATE') if 'CREATE' in t else -1
```

Comment:
`if 'CREATE' in t else -1`

I think it's better to create a small helper function to do the try-except ValueError than scan the string twice.

Reply:
hows this? 3baf73fba34230c2398f7c794e4710ca5ec2bdb4

Reply:
Nice 👍